Add documentation about S3 addon provider in Helm Broker (#5572)
kyma-project/kyma: docs/helm-broker/03-03-create-addons-repo.md

          namespace: stage
  ```

  </details>
  <details>
  <summary>
  S3
  </summary>

>   The S3 protocol requires key and secret to authorize in your bucket, to get a key/secret login to the AWS [console](https://console.aws.amazon.com),

Reviewer: ```suggestion
The S3 protocol requires a key and a Secret to authenticate with your bucket. To get a key and a Secret, log in to the AWS [console](https://console.aws.amazon.com),
```

>   select `My Security Credentials` and go to `Access keys` tab where you can create new access key.

Reviewer: ```suggestion
  select **My Security Credentials**, and go to **Access keys** tab where you can create a new access key.
```

>   > **NOTE:** For more information about security credentials and access key see [this](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) 

Reviewer: ```suggestion
  > **NOTE:** For more information about security credentials and access key, read [this](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) document. 
```

>   

Reviewer: ```suggestion
Follow these steps to secure your S3 addons repository with basic authentication credentials:
```

>   Create a Secret resource with S3 credentials

Reviewer: ```suggestion
1. Create a Secret resource with S3 credentials:
```

  ```bash
    echo -n 'AWS_KEY' > ./key.txt
    echo -n 'AWS_SECRET' > ./secret.txt
    
    kubectl create secret generic aws-auth --from-file=aws_key=./key.txt --from-file=aws_secret=./secret.txt
  ```
  

>   Define a URL with the required key/secret fields:

Reviewer: ```suggestion
2. Define a URL with the required key and Secret fields:
```
